src/Model/PaymentCard.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace ThePay\ApiClient\Model;
+
+use DateTime;
+use ThePay\ApiClient\Utils\Json;
+
+class PaymentCard
+{
+    public const TYPE_CREDIT = 'credit';
+    public const TYPE_DEBIT = 'debit';
+    public const TYPE_PREPAID = 'prepaid';
+    public const TYPE_UNKNOWN = 'unknown';
+
+    private string $number;
+    private ?DateTime $expiration_date;
+    private ?string $brand;
+    private string $type;
+
+    /**
+     * @param string|array<string,string> $values Json in string or associative array
+     */
+    public function __construct($values)
+    {
+        $data = is_array($values) ? $values : Json::decode($values, true);
+
+        $this->number = $data['number'];
+        $this->expiration_date = $data['expiration_date'] !== null ? new DateTime($data['expiration_date']) : null;
```

Comment:
I'm just thinking if expiration_date contents date of expiration, it'll need to set time to `23:59:59.999999`. Isn't that right?

Reply:
yeah actually good point, but I need to set even last day of the month, because api returns date in format Y-m